feedback-module/public/index.html

```diff
@@ -28,7 +28,12 @@
   </head>
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
-    <div id="feedback-widget"></div>
+    <div
+      id="feedback-widget"
+      lang="es"
```

Comment:
Is this supposed to be hardcoded?

Reply:
This was moreso just for testing purposes— since this index.html file doesn't get copied over in the make build it shouldn't make a big difference

Reply:
Ok sounds good!